zebrad/tests/acceptance.rs

```diff
@@ -517,6 +502,37 @@ fn config_test() -> Result<()> {
     // Check that an older stored configuration we have for Zebra works
     stored_config_works()?;
 
+    // Runs `zebrad` serially to avoid potential port conflicts
+    app_no_args()?;
+
+    Ok(())
+}
+
+/// Test that `zebrad` runs the start command with no args
+fn app_no_args() -> Result<()> {
+    let _init_guard = zebra_test::init();
+
+    // start caches state, so run one of the start tests with persistent state
+    let testdir = testdir()?.with_config(&mut persistent_test_config()?)?;
+
+    let mut child = testdir.spawn_child(args![])?;
+
+    // Run the program and kill it after a few seconds
+    std::thread::sleep(LAUNCH_DELAY);
+    child.kill()?;
```

Comment:
Update based on test API changes from another PR

```suggestion
    child.kill(true)?;
```